addons/website/tests/test_configurator_translation.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from unittest.mock import patch
+
+import odoo.tests
+from odoo.addons.iap.tools.iap_tools import iap_jsonrpc_mocked
+from odoo.tools.misc import mute_logger
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestConfigurator(odoo.tests.HttpCase):
+
+    def _theme_upgrade_upstream(self):
+        # patch to prevent module install/upgrade during tests
+        pass
+
+    def setUp(self):
+        super().setUp()
+
+        def iap_jsonrpc_mocked_configurator(*args, **kwargs):
+            endpoint = args[0]
+            params = kwargs.get('params', {})
+            language = params.get('lang', 'en_US')
+            if endpoint.endswith('/api/website/1/configurator/industries'):
+                if language == 'fr_FR':
+                    return {"industries": [
+                        {"id": 1, "label": "abbey in fr"},
+                        {"id": 2, "label": "aboriginal and torres strait islander organisation in fr"},
+                        {"id": 3, "label": "aboriginal art gallery in fr"},
+                        {"id": 4, "label": "abortion clinic in fr"},
+                        {"id": 5, "label": "abrasives supplier in fr"},
+                        {"id": 6, "label": "abundant life church in fr"}]}
+                else:
+                    return {"industries": [
+                        {"id": 1, "label": "abbey"},
+                        {"id": 2, "label": "aboriginal and torres strait islander organisation"},
+                        {"id": 3, "label": "aboriginal art gallery"},
+                        {"id": 4, "label": "abortion clinic"},
+                        {"id": 5, "label": "abrasives supplier"},
+                        {"id": 6, "label": "abundant life church"}]}
+            elif '/api/website/2/configurator/recommended_themes' in endpoint:
+                return []
+            elif '/api/website/2/configurator/custom_resources/' in endpoint:
+                return {'images': {}}
+
+            iap_jsonrpc_mocked()
+
+        with mute_logger('odoo.addons.base.models.ir_translation'):
+            self.env["base.language.install"].create({'lang': 'fr_FR', 'overwrite': True}).lang_install()
+        self.lang_id = self.env.ref('base.lang_fr')
+        self.fr_website_id = self.env['website'].create({'name': "Website Fr", 'default_lang_id': self.lang_id.id})
```

Comment:
```suggestion
        self.website_fr = self.env['website'].create({'name': "Website Fr", 'default_lang_id': self.lang_id.id})
```
Variable suffixed by `_id` are supposed to hold.. ID.. Not record.